R/util.R

```diff
@@ -83,3 +83,17 @@ find_r <- function() {
 }
 
 is_macos <- function() tolower(Sys.info()["sysname"]) == "darwin"
+
+local_dir <- function() {
```

Comment:
Can you add a comment explaining the difference between local_dir and local_data_dir?

Do we need to support both env var and option?

Reply:
Sure, can do — we don't need to I left it in more for backwards compatibility (though that's probably silly for something this new). I'm happy to pull out the options if you think they are more confusing / not worth keeping around.

Reply:
Yeah I don't think you have to worry too much about breaking APIs, especially rather internal ones like this.